src/commands/developers/EvalCommand.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable @typescript-eslint/no-unused-vars, no-eval */
+import { DefineCommand } from "../../utils/decorators/DefineCommand";
+import { CommandContext } from "../../structures/CommandContext";
+import { BaseCommand } from "../../structures/BaseCommand";
+import { createEmbed } from "../../utils/createEmbed";
+import i18n from "../../config";
+import { Message } from "discord.js";
+import { request } from "https";
+import { inspect } from "util";
+
+@DefineCommand({
+    aliases: ["evaluate", "ev", "js-exec"],
+    cooldown: 0,
+    description: i18n.__("commands.developers.eval.description"),
+    devOnly: true,
+    name: "eval",
+    usage: i18n.__("commands.developers.eval.usage")
+})
+export class EvalCommand extends BaseCommand {
+    public async execute(ctx: CommandContext): Promise<Message|void> {
+        const msg = ctx;
+        const client = this.client;
+
+        const embed = createEmbed("info")
+            .addField("Input", `\`\`\`js\n${ctx.args.join(" ")}\`\`\``);
+
+        try {
+            let code = ctx.args.slice(0).join(" ");
```

Comment:
```suggestion
            let code = ctx.args.join(" ");
```

Slicing is not needed here